manm_cs/prob_distributions/discrete/uniform_discrete_distribution.py

```diff
@@ -10,4 +10,4 @@ def __init__(self, num_values: int):
         super(UniformDiscreteDistribution, self).__init__(num_values=num_values)
 
     def sample(self, num_observations: int) -> np.array:
-        return np.random.randint(low=0, high=self.num_values - 1, size=num_observations)
+        return np.random.randint(low=0, high=self.num_values, size=num_observations)
```

Comment:
good spotting, that was indeed incorrect, as num values = 2 would have caused a discrete domain of 1